### ISSUE TITLE
Add to IPython wiki

### ISSUE THREAD

**Issue by ghost** [interactivecomputing/ijs#3]
Hi @nikhilk, and first thanks to share your project on github. I'm really interested in using ipython notebooks for javascript for my students next year, and your implementation seems quite valuable for my needs with both js on front and backend. 

Maybe you know that there is another project implementing a javascript kernel for ipython/jupyter, it's named [ijavascript](https://github.com/n-riesco/ijavascript) and it is listed in the ipython [wiki](https://github.com/ipython/ipython/wiki/IPython-kernels-for-other-languages), maybe you could add you're project to the wiki to give it more visibility.

**Comment by nikhilk** [comment]
Yes, I need to - thanks for the reminder! Will keep this issue open as a reminder. I was hoping to get my github pages in order before linking.

In the meanwhile its great to hear you see this as promising for educational scenarios. I love the idea of js across the board. Any suggestions for what would be useful, the kinds of learning scenarios you plan for would be great to hear to help shape things out.

**Comment by ghost** [comment]
Hi @nikhilk and sorry for the delay. I think interactive computing is really nice to learn programming because students can work with code almost in real time without any boilerplate. I've already used ipython notebook for this, and I'll continue to use it because python is a really nice language, however students are really interested in web development and so js is a must. 
When they work on an interactive session things need to be as simple as possible with nice outputs and concise errors output. 
I work with **svg** and **canvas**, it also would be nice to get **inputs** from the user and store easily these inputs as we can do in the ipython notebook.

**Comment by nikhilk** [comment]
Totally agree on the appeal of js.

Thanks for calling out some specifics. Some followup questions.
- SVG and canvas -- I haven't done much here, but do you envision its interesting to have a cell define an instance of an SVG container or a canvas surface, and then a bunch of other cells working against that cell, i.e. the svg/canvas element from the one cell rather than each cell creating its own svg/canvas elements?
- User inputs -- could you explain a bit more? Are you thinking about the interactive widgets in ipython? Or something else?

Related to web development, I am envisioning a web server associated with the notebook, and code within the notebook can stash some content and get a URI, or register a function defined in the notebook as a route handler, and then be able to make HTTP requests to that server from client-side script either in the notebook, or from some other app. This would make it possible to develop web APIs within the notebook - think express app implemented in a notebook. I just started, so this isn't available yet, but sharing as it seems like it would be relevant.
